JackSinnott/JECL_Ducks_Roguelike
# Dev/pure grid system

4 rooms currently being drawn to sceen. The rooms are all of different types (different .txt files used to generate them).  The fileloader just sequentially goes through each .txt file and draws for now not truely dynamic but its working. The grid draws multiple rooms based on the data passed to fileloader but isn't assimilating any of the data yet itself. That's my hurdle for next week!

Steps to test: 
1) when you run the gameplay screen you'll see 4 rooms drawn. when you rerun the game (or just reload) you should see the same 4 rooms being drawn for now.
2) Remove a laod call from gameplay.cpp and you should be able to load just three rooms with no issues!
3) if you try add a load call it will probably break just fyi.
